### DISCUSSION TITLE
New Release

### DISCUSSION THREAD

**Discussion by YangSan0622** [Apicurio/apicurio-registry#3010] [General]
Hello guys,

I am just wondering when will there be a new release? I have bugfix PR merged but it did not make it in the Sept 23rd release so I was wondering when will there be another release.

**Comment by carlesarnal** [comment]
Hi, @YangSan0622, we have plans to do a release "soon" with soon meaning before the end of the year.

Thanks

**Comment by EricWittmann** [comment]
Release was done today!
